app/_hub/kong-inc/jwt-signer/index.md

```diff
@@ -290,7 +294,7 @@ params:
       default:
       datatype: number
       description: |
-        Timeout for introspection request. XX REVIEWERS: what is the UoM? MS? S? What is a recommended value? XX
+        Timeout for introspection request.
```

Comment:
asked @bungle about this is slack TBD

Reply:
MS per @bungle. Additional information: 

my question: I'm not sure about
If both requests timeout, then the plugin runs two times the
        `config.access_token_introspection_timeout` on access token introspection.

answer: bungle  1:03 AM

this: https://github.com/ledgetech/lua-resty-http#set_timeout

it sets connection, read and write timeout for the tcp socket connection (that http client uses)

https://github.com/openresty/lua-nginx-module#tcpsocksettimeout